Internal/processProviders/processProviders.go

```diff
@@ -26,11 +26,20 @@ import (
 	"os/exec"
 	"regexp"
 	"strconv"
+	"strings"
 	s "strings"
```

Comment:
Should this line be removed?

Reply:
It's already taken care of here: https://github.com/Juniper/junos-terraform/pull/16/commits/ae86ffe6b781a1660a58cdb9de71c458a8bcedf3 

This commit was made just after the PR, but can be found above this commit.